srcDoc/release_notes.rst

```diff
@@ -70,6 +70,14 @@ and PDF documentation from the rst files in srcDoc.
 What's Fixed in Master
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Fix incorrect behavior of -unbounded with small data (#125)
```

Comment:
Fix incorrect behavior the `-unbounded` command-line option when not using large data (#125)

Reply:
Done